src/js/claims-status/containers/YourClaimsPage.jsx

```diff
@@ -118,27 +118,49 @@ class YourClaimsPage extends React.Component {
   }
 
   render() {
-    const { unfilteredAppeals, unfilteredClaims, list, pages, page, loading, show30DayNotice, route, synced } = this.props;
+    const {
+      unfilteredAppeals,
+      unfilteredClaims,
+      list,
+      pages,
+      page,
+      claimsLoading,
+      appealsLoading,
+      show30DayNotice,
+      route,
+      synced
+    } = this.props;
+
     const tabs = [
       'OpenClaims',
       'ClosedClaims'
     ];
 
     let content;
     let innerContent;
+    const bothRequestsLoaded = !claimsLoading && !appealsLoading;
+    const bothRequestsLoading = claimsLoading && appealsLoading;
+    const atLeastOneRequestLoading = claimsLoading || appealsLoading;
+    const emptyList = !list || !list.length;
 
-    if (loading) {
-      content = <LoadingIndicator message="Loading a list of your claims and appeals..." setFocus/>;
+    if (bothRequestsLoading || (atLeastOneRequestLoading && emptyList)) {
+      content = <LoadingIndicator message="Loading your claims and appeals..." setFocus/>;
```

Comment:
Does `setFocus` need to be set to true?

Reply:
I don't think so, since it's possible some claims or appeals are visible and the user is interacting with them already. I don't want to steal focus in that case.